src/middlewared/middlewared/plugins/disk_/wipe.py

```diff
@@ -37,7 +54,7 @@ def _wipe_impl(self, job, dev, mode, event):
 
             if mode == 'QUICK':
                 # Get partition info before it gets destroyed
-                disk_parts = self.middleware.call_sync('disk.list_partitions', dev)
+                disk_parts = self.get_partitions_quick(dev)
```

Comment:
We'll need to handle when this crashes (i.e. `FileNotFoundError` and `ValueError`)

Reply:
That was my original approach, but...
I used `glob` to find the files.  This doesn't generate an error or exception if nothing is found.  It just returns an empty dictionary.   To keep it lightweight I decided to not add a separate existence test.
If you think it's better, I can add a test and raise the exception when the path is missing.
Let me know.

Reply:
Sorry, I meant when you do these 2 operations.
```
       with open(f"/sys/block/{dev_name}/queue/logical_block_size", "r") as fd:
             sectsize = int(fd.read().strip())
```
It would be crazy to have it fail there, but I've seen some crazier scenarios 😄

Reply:
So something like
```
try:
    with open(blah) as f:
        sect size = int(f.bla)
except FileNotFoundError, ValueError:
    pass
except Exception:
    self.logger.error('Unexpected failure trying to blah', exc_info=True)